packages/asar.vm/asar.vm.nuspec

```
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>asar.vm</id>
+    <version>3.4.1</version>
+    <authors>@electron/asar contributors</authors>
+    <description>asar decompresses .asar archives</description>
+    <dependencies>
+      <dependency id="common.vm" version="0.0.0.20240514" />
```

`VM-Get-Category` was introduced in (version `0.0.0.20250206` of common.vm`). We need to require that version or the package installation may fail.
```suggestion
      <dependency id="common.vm" version="0.0.0.20250206" />
```

related: https://github.com/mandiant/VM-Packages/issues/1369